src/_data/data.js

```diff
@@ -322,25 +323,45 @@ function createLists(faqDir) {
 // Authors Processing
 // ============================================================================
 
-// Read and return AUTHORS.md content
-function processAuthorsFile() {
-  const authorsPath = path.join(FAQ_DIR, "AUTHORS.md");
 
-  if (!fs.existsSync(authorsPath)) {
-    throw new Error(`AUTHORS.md not found at ${authorsPath}. Ensure the cache is populated.`);
+// Read and return AUTHORS.md/CONTRIBUTORS.md content
+function fetchAcknowledgementsFile(path) {
+
+  if (!fs.existsSync(path)) {
+    throw new Error(`markdown file not found at ${path}. Ensure the cache is populated.`);
   }
 
-  const rawContent = fs.readFileSync(authorsPath, "utf-8");
+  const rawContent = fs.readFileSync(path, "utf-8");
   const parsed = matter(rawContent);
   const content = parsed.content.trim();
+  
 
   if (!content) {
-    throw new Error(`AUTHORS.md at ${authorsPath} is empty or has no content after frontmatter.`);
+    throw new Error(`markdown file at ${path} is empty or has no content after frontmatter.`);
   }
 
   return content;
 }
 
+// Read, curate and compose the data for the merge of the two files
+function processAcknowledgements(authorsPath,contribPath)
+{
+  // Extract the different names list in the bodies into arrays
+  const authorsNames = extractNames(fetchAcknowledgementsFile(authorsPath));
+  const contribNames = extractNames(fetchAcknowledgementsFile(contribPath));
+
+  return {authorsNames, contribNames};
+}
+
+function extractNames(contribBody)
+{
+  // Add the g flag to take all the matches
+  nameMatches = contribBody.match(/^\*\s+(.+)$/gm);
+
+  //Remove the * in the begining
+  return names = nameMatches.map(match => match.replace(/^\*\s+/, '').trim());
+}
```

Comment:
Use clearer arg names. Don't forget to declare the constant.
```suggestion
function extractNames(content) {
  const names = content.match(/^\*\s+(.+)$/gm);

  //Remove the * in the beginning
  return names.map(name => name.replace(/^\*\s+/, '').trim());
}
```